### DISCUSSION TITLE
error at  Symfony\Bridge\ProxyManager\LazyProxy\PhpDumper\LazyLoadingValueHolderGenerator

### DISCUSSION THREAD

**Discussion by kukulis** [symfony/symfony#46953] [Q&A]
Hi,

There is an error at  Symfony\Bridge\ProxyManager\LazyProxy\PhpDumper\LazyLoadingValueHolderGenerator

instead of use Zend\Code\Generator\ClassGenerator
should be Laminas\Code\Generator\ClassGenerator

**Comment by stof** [comment]
Which version of Symfony are you using ?

**Reply by kukulis** [reply]
3.4.2

Its a legacy project.

**Reply by stof** [reply]
The 3.4 branch stopped receiving bug fixes in November 2020: https://symfony.com/releases/3.4

So if you cannot update to a maintained version, you should look into using the compatibility layer provided by the laminas project, or to stick to old versions of other packages too to keep using zend-code.

No new version of Symfony 3.4 will be done to backport the upgrade of that code.

**Reply by kukulis** [reply]
I go to the vendors directory and edit the file manually, to avoid runtime exceptions.
@stof 
Do you have any suggestions to workaround this? :)
